src/store/getters/state-getters.ts

```diff
@@ -241,14 +240,18 @@ export const isBusySaving = (state: any): boolean => {
     state.stateModel.tombstone.isFilingPaying)
 }
 
-/** Whether any correction sections have changed. */
-export const isCorrectionChanged = (state: any): boolean => {
+/** Whether any correction/alteration sections have changed. */
+export const isFilingChanged = (state: any): boolean => {
   // TODO: add other sections here
-  return (state.stateModel.peopleAndRoles.changed)
+  return (state.stateModel.peopleAndRoles.changed ||
+    state.stateModel.defineCompanyStep.changed ||
+    state.stateModel.createShareStructureStep.changed ||
+    state.stateModel.incorporationAgreementStep.changed)
 }
 
-/** Whether all the correction sections are valid. */
-export const isCorrectionValid = (state: any): boolean => {
-  // TODO: add other sections here
+/** Whether any correction/alteration sections have changed. */
+export const isFilingValid = (state: any): boolean => {
+  // Add sections that can have only invalid data like people and roles and share structure
+  // Define company, Agreement Type wont allow saving and invalid state to the store
```

Comment:
Staff payment validity should go here I think.

Reply:
As explained in the other comment, if we want to allow invalid filing to be saved I can remove the check from save action buttons and payment validity can be here.

Reply:
Can payment validity be here, whether or not we allow invalid filing to be saved?

I think the only difference would be whether you check "isFilingValid" for the save buttons.

If I misunderstand, please explain. Thx!

Reply:
If staff payment validity is a check on the filing validity as a whole, then i believe it should be conditional based on the 'staff' role. Unless that's something we want to think about later. To be honest, i'm not sure what the plan is for client driven corrections..

Reply:
I think we can do that when we do self serve alteration.

Reply:
Resolving this conversation... I'll look at your update code ASAP.